### ISSUE TITLE
[SAGE] View Item

### ISSUE THREAD

**Issue by devangm** [TAMULib/SAGE#565]
**URL:** https://library.tamu.edu/discovery/discovery-context/berger-cloonan/aHR0cHM6Ly9hcGkubGlicmFyeS50YW11LmVkdS9mY3JlcG8vcmVzdC8zYi82Zi9jMy8yNS8zYjZmYzMyNS1mNmNhLTQxZDgtYjkxZS04YzVkYjNiZThjMTMvYmVyZ2VyX2Nsb29uYW5fb2JqZWN0cy8zOQ==
**Violation:** AA: Contrast (Minimum)
**Issues:** Very low contrast
**Notes:** Link on breadcrumb is very light

**URL:**  https://library.tamu.edu/discovery/discovery-context/berger-cloonan/aHR0cHM6Ly9hcGkubGlicmFyeS50YW11LmVkdS9mY3JlcG8vcmVzdC8zYi82Zi9jMy8yNS8zYjZmYzMyNS1mNmNhLTQxZDgtYjkxZS04YzVkYjNiZThjMTMvYmVyZ2VyX2Nsb29uYW5fb2JqZWN0cy8zOQ==
**Violation:** A: Info and Relationships
**Issues:** Skipped heading level
**Notes:** About this item heading skipped heding level

**URL:**  https://library.tamu.edu/discovery/discovery-context/berger-cloonan/aHR0cHM6Ly9hcGkubGlicmFyeS50YW11LmVkdS9mY3JlcG8vcmVzdC8zYi82Zi9jMy8yNS8zYjZmYzMyNS1mNmNhLTQxZDgtYjkxZS04YzVkYjNiZThjMTMvYmVyZ2VyX2Nsb29uYW5fb2JqZWN0cy8zOQ==
**Violation:** 
**Issues:**  Very small text
**Notes:** Current item, reource, related' very small text

**Comment by qtamu** [comment]
The text within Mirador might not be able to be changed within the SAGE app itself.
